.travis.yml

```diff
@@ -1,29 +1,19 @@
+dist: bionic
 language: python
-dist: bionic   # required for Python >= 3.7
```

Comment:
Why remove the comment and move the line? I added the comment for a reason.

Reply:
That comment was important when __trusty__ was the default distro on Travis CI because trusty did not support Python 3.7.

But now that xenial is the default distro, the comment is no longer needed because both xenial and bionic fully support Python 3.7  https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment

Reply:
Reasoning like this should be part of the commit message.